### ISSUE TITLE
[Bug]: Fail to parse "\limits"  in mathtext

### ISSUE THREAD

**Issue by ChenZiHong-Gavin** [matplotlib/matplotlib#28051]
### Bug summary

I encountered an issue while using the mathtext module in matplotlib. The problem arises when trying to parse LaTeX expressions containing the "\limits" command. The "\limits" command is used in LaTeX to specify that certain mathematical operators should have their subscripts and superscripts displayed above and below, especially when used with "\int".

For example,
`\int \limits _ { v } ^ { \mu e } e d A`
get
![image](https://github.com/matplotlib/matplotlib/assets/58508660/bb0e76c3-65cb-46ed-857b-4f7808888ebf)
and `\int_ { v } ^ { \mu e } e d A`
get
![image](https://github.com/matplotlib/matplotlib/assets/58508660/02abf05a-c598-4b1c-aa00-5a6478e36101)

### Code for reproduction

```Python
from matplotlib import mathtext
latex_text = r"$\int\limits_{ v } ^ { \mu e } e d A$"
mathtext.math_to_image(latex_text, 'test.png')
```

### Actual outcome

ValueError: 
\int\limits_{ v } ^ { \mu e } e d A
    ^
ParseFatalException: Unknown symbol: \limits, found '\'  (at char 4), (line:1, col:5)

### Expected outcome

![image](https://github.com/matplotlib/matplotlib/assets/58508660/059d414f-a9c4-46c0-8cdb-564d291f3c9a)

### Operating system

Windows

### Matplotlib Version

3.8.4

### Matplotlib Backend

module://matplotlib_inline.backend_inline

### Python version

3.9.13

### Jupyter version

1.0.0

### Installation

pip

**Comment by oscargus** [comment]
Yes, `\limits` is not supported. I guess there are two possible solutions with varying difficulty here:
1. Get rid of the parsing error, but ignore the meaning of `\limits` (easier)
2. Parse `\limits` correctly and position the limits as expected (harder).

(I changed this to an enhancement request and modified the title accordingly.)

**Comment by clementgilli** [comment]
```\limits``` is parsed as an unknown symbol, so I think we have to parse it like something to pass. But if you just pass when you have this symbol you will get the following error :
```
File "/matplotlib/lib/matplotlib/_mathtext.py", line 1237, in kern
    kerning_distance = elem.get_kerning(next)
AttributeError: 'str' object has no attribute 'get_kerning'
```
Maybe there is already a parsing class which allows to "pass" the parsing of this symbol, but I can't find it.

**Comment by Ayush2029** [comment]
Hi! I'd like to work on this issue. Could it be assigned to me?

**Comment by QuLogic** [comment]
We don't assign issues; commenting and/or opening a PR is enough.

**Comment by ahmed5145** [comment]
I've opened a PR that adds parsing for `\limits` and `\nolimits`: https://github.com/matplotlib/matplotlib/pull/32333

It reuses mathtext's existing over/under layout (the machinery already used for `\sum`, `\prod`, `\lim`, etc.): `\limits` forces the operator's sub/superscripts above and below it, `\nolimits` forces them to the side. So the issue's example `$\int\limits_{v}^{\mu e} e\, dA$` now renders with the bounds stacked on the integral sign. Both commands also parse when nothing follows (e.g. `$\int\limits$`).
